src/Storages/MergeTree/IntegerCodecTrait.h

```diff
@@ -0,0 +1,232 @@
+#pragma once
```

Comment:
Can we move all of the stuff in this file into MergeTreeIndexTextPostingListCodec.cpp?